ecs-cli/modules/clients/aws/ssm/client.go

```diff
@@ -55,7 +55,7 @@ func NewSSMClient(commandConfig *config.CommandConfig) Client {
 
 func (c *ssmClient) GetRecommendedECSLinuxAMI() (*AMIMetadata, error) {
 	response, err := c.client.GetParameter(&ssm.GetParameterInput{
-		Name: aws.String(amazonLinuxRecommendedParameterName),
+		Name: aws.String(amazonLinux2RecommendedParameterName),
```

Comment:
Woohoo!

Are there any other tangential changes (such as default user data scripts) that could be required with switching to AL2?

Reply:
@jahkeup not sure honestly, you tell me 😛

This is our default user data script template:
```
#!/bin/bash
echo ECS_CLUSTER=%s >> /etc/ecs/ecs.config
```